pkg/flavors/postgres/postgres.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package postgres
+
+import (
+	"fmt"
+
+	valid "github.com/asaskevich/govalidator"
+	"github.com/tetratelabs/getenvoy/pkg/flavors"
+)
+
+// Define template parameter names
+const endpoint string = "endpoint"
+const inport string = "inport"
+
+// Flavor implements flavor.FlavorConfigTemplate interface
+// and stores config data specific to Postgres template.
+type Flavor struct {
+	// Location of the postgres server
+	endpoint string
+	// Envoy's listener port
+	inport string
```

Comment:
These fields should be exported to parse them by `text/template` packages.

Reply:
I just noticed that too. Thanks. I had originally Endpoint and InPort, but changed to lowercase because of linter.